src/pages/product-list-page/product-sort/product-sort.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { TextField } from '@material-ui/core';
+import useStyles from './product-sort.styles';
+import ProductsPerPage from '../products-per-page';
+import {
+  setSortByDate,
+  setSortByPrice,
+  setSortByRate,
+  setSortByPopularity
+} from '../../../redux/products/products.actions';
+import {
+  SORT_BY_SELECT_OPTIONS,
+  SORT_BY_TEXT
+} from '../../../translations/product-list.translations';
+
+const ProductSort = () => {
+  const { language } = useSelector(({ Language: { language } }) => ({
+    language
+  }));
+  const styles = useStyles();
+  const dispatch = useDispatch();
+
+  const selectHandler = (e) => {
+    const { name, value } = JSON.parse(e.target.value);
+
+    if (name === 'sortAsc' || name === 'sortDesc') {
```

Comment:
Please, move `sortAsc` and `sortDesc` to the constants in the config file

Reply:
fixed